src/routes/propertyRoutes.js

```diff
@@ -8,11 +8,179 @@ import { verifyToken } from '../middleware/auth';
 
 const propertyRouter = express.Router();
 
+/**
+ * @swagger
+ * components:
```

Comment:
This endpoint requires bearer token, you should specify that on the API docs, you can check the docs on how to specify the token field.